client/my-sites/checklist/checklist-show/index.jsx

```diff
@@ -60,6 +62,44 @@ class ChecklistShow extends PureComponent {
 		}
 	};
 
+	renderHeader( completed ) {
+		const shareTo = [ 'facebook', 'twitter', 'linkedin', 'google-plus', 'pinterest' ];
+
+		if ( ! completed ) {
+			return (
+				<FormattedHeader
+					headerText="Welcome back!"
+					subHeaderText="Let’s get your site ready for its debut with a few quick setup steps."
+				/>
+			);
+		}
+
+		return (
+			<Fragment>
+				<img
+					src="/calypso/images/signup/confetti.svg"
+					aria-hidden="true"
+					className="checklist-show__confetti"
+				/>
+				<FormattedHeader
+					headerText="Conguratulations!"
```

Comment:
`Conguratulations` should be Congratulations

Reply:
😱